Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -480,6 +480,30 @@ mode would look as follows:
 
 .. _XDP acceleration:
 
+Socket LoadBalancer Bypass in Pod Namespace
+*******************************************
+
+Cilium has built-in support for bypassing the socket-level LoadBalancer and falling back
+to the tc LoadBalancer when a custom redirection/operation relies on the original ClusterIP
+within pod namespace (e.g., Istio side-car).
+
+Setting ``loadBalancer.hostNamespaceOnly=true`` enables this bypassing mode. When enabled,
+this circumvents socket rewrite in the ``connect()`` and ``sendmsg()`` syscall bpf hook and
+will pass the original packet to next stage of operation (e.g., stack in
+``per-endpoint-routing`` mode) and re-enables service lookup in the tc bpf program.
+
+A Helm example configuration in a kube-proxy-free environment with socket LB bypass
+looks as follows:
+
+.. parsed-literal::
+
+    helm install cilium |CHART_RELEASE| \\
+        --namespace kube-system \\
+        --set tunnel=disabled \\
+        --set autoDirectNodeRoutes=true \\
+        --set kubeProxyReplacement=strick \\
```

Comment:
typo sneaked in here, shouldn't it be `strict`

Reply:
Done.